src/components/Activity/Activity.tsx

```diff
@@ -32,6 +32,10 @@ const ActionsPredecescence: Record<TokenActionType, number> = {
   AUCTION_BID                   : 4,
   AUCTION_CANCELLED             : 4,
   AUCTION_FULFILLED             : 4,
+  ARTICLE_MINTED                : 0,
+  ARTICLE_EDITIONS_TRANSFERED   : 1,
+  ARTICLE_METADATA_UPDATED      : 1,
+  ARTICLE_METADATA_LOCKED       : 20,
```

Comment:
@ciphrd wasn't 100% sure about this one.

Reply:
I think it's obsolete now I'll take a deeper look later, not sure